SignalServiceKit/src/Messages/OWSBlockingManager.m

```diff
@@ -201,9 +202,9 @@ - (void)handleUpdate:(BOOL)sendSyncMessage
             [self saveSyncedBlockedPhoneNumbers:blockedPhoneNumbers];
         }
 
-        [[NSNotificationCenter defaultCenter] postNotificationName:kNSNotificationName_BlockedPhoneNumbersDidChange
-                                                            object:nil
-                                                          userInfo:nil];
+        [[NSNotificationCenter defaultCenter] postNotificationNameAsync:kNSNotificationName_BlockedPhoneNumbersDidChange
+                                                                 object:nil
+                                                               userInfo:nil];
```

Comment:
Similarly used by various components to respond to async event. Safe to post later.